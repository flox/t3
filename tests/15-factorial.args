-- sh -c 'for i in $(seq 1 15); do echo "line $i to stdout"; sleep 0.01; for j in $(seq 1 $i); do echo "line $j to stderr" 1>&2; sleep 0.01; done; done'
