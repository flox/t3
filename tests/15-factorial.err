line 1 to stderr
line 1 to stderr
line 2 to stderr
line 1 to stderr
line 2 to stderr
line 3 to stderr
line 1 to stderr
line 2 to stderr
line 3 to stderr
line 4 to stderr
line 1 to stderr
line 2 to stderr
line 3 to stderr
line 4 to stderr
line 5 to stderr
line 1 to stderr
line 2 to stderr
line 3 to stderr
line 4 to stderr
line 5 to stderr
line 6 to stderr
line 1 to stderr
line 2 to stderr
line 3 to stderr
line 4 to stderr
line 5 to stderr
line 6 to stderr
line 7 to stderr
line 1 to stderr
line 2 to stderr
line 3 to stderr
line 4 to stderr
line 5 to stderr
line 6 to stderr
line 7 to stderr
line 8 to stderr
line 1 to stderr
line 2 to stderr
line 3 to stderr
line 4 to stderr
line 5 to stderr
line 6 to stderr
line 7 to stderr
line 8 to stderr
line 9 to stderr
line 1 to stderr
line 2 to stderr
line 3 to stderr
line 4 to stderr
line 5 to stderr
line 6 to stderr
line 7 to stderr
line 8 to stderr
line 9 to stderr
line 10 to stderr
line 1 to stderr
line 2 to stderr
line 3 to stderr
line 4 to stderr
line 5 to stderr
line 6 to stderr
line 7 to stderr
line 8 to stderr
line 9 to stderr
line 10 to stderr
line 11 to stderr
line 1 to stderr
line 2 to stderr
line 3 to stderr
line 4 to stderr
line 5 to stderr
line 6 to stderr
line 7 to stderr
line 8 to stderr
line 9 to stderr
line 10 to stderr
line 11 to stderr
line 12 to stderr
line 1 to stderr
line 2 to stderr
line 3 to stderr
line 4 to stderr
line 5 to stderr
line 6 to stderr
line 7 to stderr
line 8 to stderr
line 9 to stderr
line 10 to stderr
line 11 to stderr
line 12 to stderr
line 13 to stderr
line 1 to stderr
line 2 to stderr
line 3 to stderr
line 4 to stderr
line 5 to stderr
line 6 to stderr
line 7 to stderr
line 8 to stderr
line 9 to stderr
line 10 to stderr
line 11 to stderr
line 12 to stderr
line 13 to stderr
line 14 to stderr
line 1 to stderr
line 2 to stderr
line 3 to stderr
line 4 to stderr
line 5 to stderr
line 6 to stderr
line 7 to stderr
line 8 to stderr
line 9 to stderr
line 10 to stderr
line 11 to stderr
line 12 to stderr
line 13 to stderr
line 14 to stderr
line 15 to stderr
