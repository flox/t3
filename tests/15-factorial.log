[38;5;141m[0mline 1 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 1 to stderr[0m
[38;5;141m[0mline 2 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 1 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 2 to stderr[0m
[38;5;141m[0mline 3 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 1 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 2 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 3 to stderr[0m
[38;5;141m[0mline 4 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 1 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 2 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 3 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 4 to stderr[0m
[38;5;141m[0mline 5 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 1 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 2 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 3 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 4 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 5 to stderr[0m
[38;5;141m[0mline 6 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 1 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 2 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 3 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 4 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 5 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 6 to stderr[0m
[38;5;141m[0mline 7 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 1 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 2 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 3 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 4 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 5 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 6 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 7 to stderr[0m
[38;5;141m[0mline 8 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 1 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 2 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 3 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 4 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 5 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 6 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 7 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 8 to stderr[0m
[38;5;141m[0mline 9 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 1 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 2 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 3 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 4 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 5 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 6 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 7 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 8 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 9 to stderr[0m
[38;5;141m[0mline 10 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 1 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 2 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 3 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 4 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 5 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 6 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 7 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 8 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 9 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 10 to stderr[0m
[38;5;141m[0mline 11 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 1 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 2 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 3 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 4 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 5 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 6 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 7 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 8 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 9 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 10 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 11 to stderr[0m
[38;5;141m[0mline 12 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 1 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 2 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 3 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 4 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 5 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 6 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 7 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 8 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 9 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 10 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 11 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 12 to stderr[0m
[38;5;141m[0mline 13 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 1 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 2 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 3 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 4 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 5 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 6 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 7 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 8 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 9 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 10 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 11 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 12 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 13 to stderr[0m
[38;5;141m[0mline 14 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 1 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 2 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 3 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 4 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 5 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 6 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 7 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 8 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 9 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 10 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 11 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 12 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 13 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 14 to stderr[0m
[38;5;141m[0mline 15 to stdout[0m
[38;5;141m[0m[1m[38;5;130mline 1 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 2 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 3 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 4 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 5 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 6 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 7 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 8 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 9 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 10 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 11 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 12 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 13 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 14 to stderr[0m
[38;5;141m[0m[1m[38;5;130mline 15 to stderr[0m
