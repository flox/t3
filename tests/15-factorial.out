line 1 to stdout
line 2 to stdout
line 3 to stdout
line 4 to stdout
line 5 to stdout
line 6 to stdout
line 7 to stdout
line 8 to stdout
line 9 to stdout
line 10 to stdout
line 11 to stdout
line 12 to stdout
line 13 to stdout
line 14 to stdout
line 15 to stdout
